core/server/worker/src/test/java/alluxio/worker/block/DefaultBlockWorkerTest.java

```diff
@@ -176,9 +176,15 @@ public void commitBlockOnRetry() throws Exception {
   public void createBlock() throws Exception {
     long blockId = mRandom.nextLong();
     long sessionId = mRandom.nextLong();
-    long initialBytes = 1;
+    long initialBytes = 1L;
     String path = mBlockWorker.createBlock(sessionId, blockId, 0, "", initialBytes);
-    assertTrue(path.startsWith(mMemDir)); // tier 0 is mem
+    assertTrue(path.startsWith(mMemDir));
+
+    long blockId2 = mRandom.nextLong();
+    long initialBytes2 = 1L * Constants.GB;
```

Comment:
In this test, try not to set a large initial bytes number. This test will actually create a block store. With the large initial bytes, the worker will actually try to allocate 1GB or 2GB in the system. When this test is running in a env with little disk space available, this test may be failed.

All the unit tests are better to be designed as light-weighted tests

Reply:
done